### DISCUSSION TITLE
i am confused where to start ( bought new A15 gaming laptop

### DISCUSSION THREAD

**Discussion by luvdhaval** [seerge/g-helper#1501] [Q&A] [answered]
hello guys my self Dhaval from india 
* i have pruchased this new laptop ASUS TUF Gaming A15 FA577NU-LP082W 
its now 10 days i am using after some new updates of nvidia and AS dont know i have messed something bsod started in my laptop after 5 to 6 days use 
* as of now i have reinstalled windows 11 again and updated everything again and guess what next day i was just on login screen and was just chatting on mobile even before login it carshed to bsod 
* tried to find error first i thought AC is culprit then found nvidia driver messed so i downgraded now not seen bsod for whole day today 
as you can see below screenshots 

![image](https://github.com/seerge/g-helper/assets/71142419/888328d9-6676-4968-8eeb-fba75727c8ef)
the above screenshot is of AC and the settings 

![image](https://github.com/seerge/g-helper/assets/71142419/e98fc661-ad28-4e68-85d2-6d21e16c6b93)
this is nvidia setting 

![image](https://github.com/seerge/g-helper/assets/71142419/c8ba2f04-b5b5-42ea-8849-d65337e772bf)
why this power mode is balanced showing where as in below screenshot its silence

![image](https://github.com/seerge/g-helper/assets/71142419/d5fe4009-350e-4247-baf9-422053d3a254)

as i have shared everything 
my question is as i am confused of this everything because i have used dell 1545 laptop for 13 years and nver know all about this new generation laptops searched youtube and all but not getting this new things how it works as i have purchased this for gaming and for normal use for daily task 

* so i want to know that when i do gaming i want full performance with mux switch when on power mode and also on battery when i disconnect charging at 85 to 90% charge but settings should remain high performance as i play game at night for 1 to 2 hours only 

and when i want to do normal task i want to use eco mode or balanced without that dgpu or nvidia on basic gpu to save battery 
so please help as people are here a lot experienced and let me know settings what to use if i switch to ghelper so there is no issue in warranty of laptop if i use ghelper 

p;lease guys help as my friend told me to buy this but i dont know i will face these many problems which i never faced a single problem for 13 years in old laptop and i am sorry i dont know where and in which topic should i post this

**Comment by Kikimahe** [comment]
hi,

Install Ghelper, it won't break your lapatop and you won't lose the warranty of course! :)

Download it : 
- On Windows select the "balanced" power plan.
- On Ghelper select CPU and GPU modes as you wish.

Ghelper can work with Armory Crate installed. 
But it is advisable to uninstall Armory Crate once Ghelper is installed to avoid conflicts using both software at the same time.
Use [the ASUS Armory Uninstall Tool
](https://dlcdnets.asus.com/pub/ASUS/mb/14Utilities/Armoury_Crate_Uninstall_Tool.zip?model=armoury%20crate)

**Reply by luvdhaval** [reply]
thank you so much sir ok sir i will uninstall that Armoury crate and sir, 
* what should i keep settings in nvidia app for graphics as i dont want to keep using nvidia graphics in background when in eco mode does it stops or not ?
* currently when on battery it was on silent but when i plugged in power it switches to high performance plans then when i install ghelper does this remain on balanced when i plug unplug power ? on windows setting
* i have read somewhere is the armour crate remover remove its all fine i dont have to do anything post that some say some files remains or naything we dont have to do anything manual to remove right sir ??
and 
* what about MUX switch with ghelper it auto switch for mux when on charging for gaming or it works with  battery ??

**Reply by Kikimahe** [reply, marked as answer]
- **GPU eco mode** temporarily disables your graphics card. You will notice that it is no longer available in Device Manager. You don't need to change anything in the NVIDIA application.
- Yes Ghelper will automatically switch to balanced mode when it is reconnected. Test Ghelper's CPU modes yourself and see what's happening in Windows/System/battery and power. ;-)
- Let the uninstallation tool do the job. You don't have to do anything else.
- Your mux switch will work as before. It will switch to the Nvidia graphics card when it needs it.
